src/apps/filters/h3.c

```diff
@@ -35,47 +35,51 @@
 #include "h3Index.h"
 #include "utility.h"
 
+#define PARSE()                                                             \
+    if (parseArgs(argc, argv, sizeof(args) / sizeof(Arg *), args, &helpArg, \
+                  args[0]->helpText)) {                                     \
+        return E_SUCCESS;                                                   \
+    }
+
 bool has(char *subcommand, int level, char *argv[]) {
     return strcasecmp(subcommand, argv[level]) == 0;
 }
 
-bool cellToLatLngCmd(int argc, char *argv[]) {
-    Arg cellToLatLngArg = {
-        .names = {"cellToLatLng"},
-        .required = true,
-        .helpText = "Convert an H3 cell to a WKT POINT coordinate",
-    };
-    Arg helpArg = ARG_HELP;
+Arg helpArg = ARG_HELP;
+Arg cellToLatLngArg = {
+    .names = {"cellToLatLng"},
+    .helpText = "Convert an H3 cell to a WKT POINT coordinate",
+};
+Arg latLngToCellArg = {
+    .names = {"latLngToCell"},
+    .helpText = "Convert degrees latitude/longitude coordinate to an H3 cell.",
+};
+Arg cellToBoundaryArg = {
+    .names = {"cellToBoundary"},
+    .helpText = "Convert an H3 cell to a WKT POLYGON defining its boundary",
+};
```

Comment:
Total nit: I'd put these next to the functions that implement them, but 🤷

Reply:
I waffled back and forth on that. As long as the `generalHelp` and `main` functions are the last in the file, that should work just fine, but it might be less obvious *why* they aren't defined within their function body, so I thought grouping them together as a collection of global variables would at least get people to pause before immediately wanting to refactor it and accidentally break the `generalHelp` function.

I was also thinking of turning the sub-command function declaration into a macro that makes these global variables at the same time it makes the sub-command function, but I don't know how "magic" we want the syntax in this file to be, so I didn't go that route.